### ISSUE TITLE
The text "Select destination dynamically" is a little confusing

### ISSUE THREAD

**Issue by gqqnbig** [greenshot/greenshot#567]
https://github.com/greenshot/greenshot/blob/245f6f261bb49a7ba6f51cd6de488af411a1bd6f/src/Greenshot/Languages/language-en-US.xml#L226

In the destination menu, there is an option called "Select destination dynamically". For me, the word "dynamically" sounds like the tool itself will choose a destination. But in fact, this option means users have to select a destination every time.

I think the text can be made more user-friendly. Would you prefer the following text?

- Select destination every time
- Prompt users to select a destination
- Show destination picker

**Comment by jamesaepp** [comment]
I agree with the merit of this issue.

**Comment by danrhodes** [comment]
I agree with this. Ever since using GS, this should be worded differently, as it sounds like GS will choose the destination.

**Comment by danrhodes** [comment]
I have created a pull request to aid in user understanding across all supported languages.

**Comment by danrhodes** [comment]
@jklingen merged a PR to amend the wording it is now much clearer - "Ask for destination every time". Please update to one of the 1.4 pre-releases to test.
